### PR TITLE
Connect environment variables for the ddv flutter build script

### PR DESCRIPTION
**Pull request by sebastian-aranda**
This PR adjuts the `scripts/build-flutter-app.sh` in order to connect environment variables set on the different deployments.

### CODE REVIEW

File: scripts/build-flutter-app.sh [hunk 1]
```diff
@@ -5,15 +5,13 @@ set -e
 # Display each command as it's run.
 set -x
 
-DEPLOY_BRANCH=deploy-slac
-
 cd /usr/src/rubintv
 git clone --single-branch --branch $DEPLOY_BRANCH https://github.com/lsst-sitcom/rubin_chart
 git clone --single-branch --branch $DEPLOY_BRANCH https://github.com/lsst-ts/rubintv_visualization ./ddv
```
Comment (ugyballoons) [inline scripts/build-flutter-app.sh:10]: If this is going to be set by environment var, would it be useful to have a more specified name, i.e. `DDV_DEPLOY_BRANCH`?

File: scripts/build-flutter-app.sh [hunk 2]
```diff
@@ -5,15 +5,13 @@ set -e
...
 
 # Base HREF must be bookended by "/".
-DDV_BASE_HREF=/rubintv/ddv/
-CLIENT_WS_ADDRESS=rubintv/ws/ddv
+CLIENT_WS_ADDRESS=${CLIENT_WS_ADDRESS:-rubintv/ws/ddv}
```
Comment (ugyballoons) [inline scripts/build-flutter-app.sh:13]: And I guess the same here?

**Review by ugyballoons — APPROVED** [review]
LGTM 👍

File: scripts/build-flutter-app.sh (hunk 1 above)
Comment (sebastian-aranda) [reply scripts/build-flutter-app.sh:10]: good idea! that would be better. I'll make the change :+1:.

File: scripts/build-flutter-app.sh (hunk 2 above)
Comment (sebastian-aranda) [reply scripts/build-flutter-app.sh:13]: Agree :+1:.

**Pull request merged**